lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -646,7 +647,8 @@ describe('GatherRunner', function() {
       const error = GatherRunner.getPageLoadError(url, [mainRecord]);
       assert.equal(error.message, 'FAILED_DOCUMENT_REQUEST');
       assert.equal(error.code, 'FAILED_DOCUMENT_REQUEST');
-      assert.ok(/^Lighthouse was unable to reliably load/.test(error.friendlyMessage));
+      i18n.replaceIcuMessageInstanceIds(error, 'en-US');
```

Comment:
Should we assert off the translated text, or on the pre-translation text?

Reply:
we have a helper method for this, `toBeDisplayString` https://github.com/GoogleChrome/lighthouse/blob/541e0f029436ce30d4a524dd6c9ff60f44ca7eae/lighthouse-core/test/test-utils.js#L13-L29

usage: `expect(error.friendlyMessage).toBeDisplayString(/Lighthouse was.../)`

Reply:
:+1: :+1: :+1: TIL